### DISCUSSION TITLE
Deploy Nuxt Front End on Netlify and Strapi on Digital Ocean

### DISCUSSION THREAD

**Discussion by marvthedev** [strapi/strapi#8198] [Legacy Topics]
Is it possible to deploy a nuxt front end on netlify with strapi hosted on digital ocean? I would like to use a subdomain like admin.domain.com for strapi and domain.com will be front end hosted on netlify.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
